src/docformatter/configuration.py

```diff
@@ -127,22 +127,51 @@ def do_parse_arguments(self) -> None:
             default=self.flargs_dct.get("exclude", None),
             help="in recursive mode, exclude directories and files by names",
         )
-        self.parser.add_argument(
-            "--wrap-summaries",
-            default=int(self.flargs_dct.get("wrap-summaries", 79)),
-            type=int,
-            metavar="length",
-            help="wrap long summary lines at this length; "
-            "set to 0 to disable wrapping (default: 79)",
+        black_group = self.parser.add_mutually_exclusive_group()
+        black_group.add_argument(
+            "--pre-summary-space",
+            action="store_true",
+            default=self.flargs_dct.get("pre-summary-space", "false").lower()
+            == "true",
+            help="add a space after the opening triple quotes "
+                 "(default: False)",
         )
-        self.parser.add_argument(
-            "--wrap-descriptions",
-            default=int(self.flargs_dct.get("wrap-descriptions", 72)),
-            type=int,
-            metavar="length",
-            help="wrap descriptions at this length; "
-            "set to 0 to disable wrapping (default: 72)",
+        black_group.add_argument(
+            "--black",
+            action="store_true",
+            default=self.flargs_dct.get("black", "false").lower()
+            == "true",
+            help="make formatting compatible with standard black options "
+                 "(default: False)",
         )
+        args = self.parser.parse_known_args()[0]
+        if args.black:
```

Comment:
If we do this,
```
if args.black:
    _wrap_summaries = 88
    _wrap_descriptions = 88
else:
    _wrap_summaries = 79
    _wrap_descriptions = 72
```
and then use these local variables when we set the defaults

`default=int(self.flargs_dct.get("wrap-summaries", _wrap_summaries)),`
`default=int(self.flargs_dct.get("wrap-description", _wrap_descriptions)),`

the command line or config file setting override the default `--black` settings.  That should work and then all I need to do for Google and Numpy is add a new branch to the conditional.

Reply:
Looks good to me